server/logs/lib.lua

```diff
@@ -0,0 +1,86 @@
+Logs = {}
+Logs.apiUrl= "https://api.fivemanagee.com/logs"
+Logs.apiKey = ""
+
+function Logs:start()
+    Logs.apiKey = GetConvar("FIVEMANAGE_API_KEY", "")
+end
+
+-- TODO: Add context to logs
+function Logs.findContext()
+	local info = debug.getinfo(2, "n")
+
+    return info
+end
+
+function Logs:sendHttpRequest(data)
+	PerformHttpRequest(
```

Comment:
this should prob be a local function, not on the Logs table so people can't call this themselves

Reply:
Agreed.